src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -215,6 +216,19 @@ public boolean testForCubes(CubePos start, CubePos end, Predicate<Cube> cubeAllo
     @Override public Profiler getProfiler() {
         return this.theProfiler;
     }
+    
+    /**@author Foghrye4
```

Comment:
O, Satan... Do you want it like here: https://github.com/Barteks2x/CubicChunks/blob/511d57cfaafb06ef55628bf474190a158e31725d/src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld_HeightLimits.java#L85 ?